docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: PATCH updating storage
+---
+
+
+`PATCH /asset/:id -d {"storage": {"ipfs": {}}` can be used to modify any `asset` that was stored with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
```

Comment:
We should have a page [like this](https://docs.livepeer.studio/references/stream/overview) to explain each field of the asset objects (not a change for this PR, but a note for the future). We shouldn't have fields documented scattered among multiple docs, but have a single reference documentation with all of the asset fields (then we can also link to it in places where we would explain such fields).

Reply:
Agreed

Reply:
Made an issue for this https://github.com/livepeer/livepeer-studio-docs/issues/147